<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OBD Reader</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>379</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalls">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Herramientas</string>
    </property>
    <addaction name="actionConfigure"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuCalls"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionStartLog"/>
   <addaction name="actionStopLog"/>
   <addaction name="actionConfigure"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>Sobre OBDReader</string>
   </property>
   <property name="toolTip">
    <string>Informaci√≥n sobre el programa</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="OBDReader_GUI.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>Conectar</string>
   </property>
   <property name="iconText">
    <string>Conectar</string>
   </property>
   <property name="toolTip">
    <string>Conectarse al OBDReader</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="OBDReader_GUI.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>Desconectar</string>
   </property>
   <property name="toolTip">
    <string>Desconectarse del OBDReader</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="OBDReader_GUI.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>Configurar</string>
   </property>
   <property name="toolTip">
    <string>Configura el puerto OBD</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="OBDReader_GUI.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="iconText">
    <string>Salir</string>
   </property>
   <property name="toolTip">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionStartLog">
   <property name="icon">
    <iconset resource="OBDReader_GUI.qrc">
     <normaloff>:/images/start-button.png</normaloff>:/images/start-button.png</iconset>
   </property>
   <property name="text">
    <string>StartLog</string>
   </property>
   <property name="toolTip">
    <string>Empezar el log</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionStopLog">
   <property name="icon">
    <iconset resource="OBDReader_GUI.qrc">
     <normaloff>:/images/stop-button-red-md.png</normaloff>:/images/stop-button-red-md.png</iconset>
   </property>
   <property name="text">
    <string>StopLog</string>
   </property>
   <property name="toolTip">
    <string>Parar el log</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="OBDReader_GUI.qrc"/>
 </resources>
 <connections/>
</ui>
